packages/plugin-inline/src/Format.ts

```diff
@@ -2,16 +2,31 @@ import { Modifier } from '../../core/src/Modifier';
 import { Modifiers } from '../../core/src/Modifiers';
 import { Attributes } from '../../plugin-xml/src/Attributes';
 
+/**
+ * A Format might represent DOM inline nodes (a, b, strong) that were parsed or
+ * the DOM inline node (a, b, strong) that we might render.
+ */
 export class Format extends Modifier {
     htmlTag: string; // TODO: remove this reference to DOM.
+    /**
+     * The modifiers for a format is `Attribute`.
```

Comment:
No idea what that sentence is supposed to mean. If you mean that the only modifier a `Format` can have is `Attributes`, that's wrong. `Modifiers` can potentially hold any kind of information to modify a node or a format.